src/test/java/com/datadog/api/TestUtils.java

```diff
@@ -266,5 +271,38 @@ public void cleanAndSendExpectations() {
         public void resetWiremock() {
             reset();
         }
+
+        /**
+         * Returns a unique string that can be used as a title/description/summary/... of an API entity.
+         * When used in Azure Pipelines and RECORD=true or RECORD=none, it will include BuildId to enable
+         * mapping resources that weren't deleted to builds.
+         *
+         * @return unique entity name to use in tests
+         */
+        public String getUniqueEntityName() {
+            String buildId = System.getenv("BUILD_BUILDID");
```

Comment:
Where is `BUILD_BUILDID` coming from?

Reply:
This is Azure pipeline specific env value. We are using the same one in Go tests.